src/components/toast/Toast.jsx

```diff
@@ -0,0 +1,11 @@
+import './Toast.css'
+
+const Toast = () => {
+  return (
+    <div className="toast">
+      <span className="toastText">URL이 복사되었습니다</span>
```

Comment:
텍스트는 외부에서 받아야 하지 않을까요~?
```suggestion
      <span className="toastText">URL이 복사되었습니다</span>
```